character-sfx.js

```diff
@@ -91,7 +97,19 @@ class CharacterSfx {
     if (!this.player.avatar) {
       return;
     }
-
+    
+    {
+          //################################################ trace player dir ########################################
+          this.localVector.x=0;

```

Comment:
For this you can use `.set`

Reply:
[use set for dirVector](https://github.com/webaverse/app/pull/3041/commits/c4d08a1ee77177dc0de2ff6ef95ac4108373910e)